chart/gatekeeper-operator/templates/gatekeeper.yaml

```diff
@@ -0,0 +1,517 @@
+    chart: '{{ template "gatekeeper-operator.name" . }}'
+    heritage: '{{ .Release.Service }}'
+    release: '{{ .Release.Name }}'
+  name: gatekeeper-manager-rolebinding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: gatekeeper-manager-role
+subjects:
+- kind: ServiceAccount
+  name: default
+  namespace: gatekeeper-system
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  labels:
+    app: '{{ template "gatekeeper-operator.name" . }}'
+    chart: '{{ template "gatekeeper-operator.name" . }}'
+    heritage: '{{ .Release.Service }}'
+    release: '{{ .Release.Name }}'
+  name: gatekeeper-webhook-server-secret
+  namespace: gatekeeper-system
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: '{{ template "gatekeeper-operator.name" . }}'
+    chart: '{{ template "gatekeeper-operator.name" . }}'
+    control-plane: controller-manager
+    controller-tools.k8s.io: "1.0"
+    heritage: '{{ .Release.Service }}'
+    release: '{{ .Release.Name }}'
+  name: gatekeeper-controller-manager-service
+  namespace: gatekeeper-system
+spec:
+  ports:
+  - port: 443
+    targetPort: 8443
+  selector:
+    app: '{{ template "gatekeeper-operator.name" . }}'
+    chart: '{{ template "gatekeeper-operator.name" . }}'
+    control-plane: controller-manager
+    controller-tools.k8s.io: "1.0"
+    heritage: '{{ .Release.Service }}'
+    release: '{{ .Release.Name }}'
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  labels:
+    app: '{{ template "gatekeeper-operator.name" . }}'
+    chart: '{{ template "gatekeeper-operator.name" . }}'
+    control-plane: controller-manager
+    controller-tools.k8s.io: "1.0"
+    heritage: '{{ .Release.Service }}'
+    release: '{{ .Release.Name }}'
+  name: gatekeeper-controller-manager
+  namespace: gatekeeper-system
+spec:
+  replicas: {{ .Values.Replicas }}
+  resources: 
+{{ toYaml .Values.resources | indent 12 }}
+    {{- with .Values.nodeSelector }}
+      nodeSelector:
+{{ toYaml . | indent 8 }}
+    {{- end }}
+    {{- with .Values.affinity }}
+      affinity:
+{{ toYaml . | indent 8 }}
+    {{- end }}
+    {{- with .Values.tolerations }}
+      tolerations:
+{{ toYaml . | indent 8 }}
+    {{- end }}
+  selector:
+    matchLabels:
+      app: '{{ template "gatekeeper-operator.name" . }}'
+      chart: '{{ template "gatekeeper-operator.name" . }}'
+      control-plane: controller-manager
+      controller-tools.k8s.io: "1.0"
+      heritage: '{{ .Release.Service }}'
+      release: '{{ .Release.Name }}'
+  serviceName: gatekeeper-controller-manager-service
+  template:
+    metadata:
+      labels:
+        app: '{{ template "gatekeeper-operator.name" . }}'
+        chart: '{{ template "gatekeeper-operator.name" . }}'
+        control-plane: controller-manager
+        controller-tools.k8s.io: "1.0"
+        heritage: '{{ .Release.Service }}'
+        release: '{{ .Release.Name }}'
+    spec:
+      containers:
+      - args:
+        - --auditInterval=30
```

Comment:
can we have `auditInterval` and `constraintViolationsLimit` as variables? (defaulting to their default values)

Reply:
Yes @sozercan good call, thanks for the suggestion.